packages/api/index.js

```diff
@@ -4,14 +4,14 @@ let methods = require('./methods');
 let utils = require('./utils');
 let actualApp;
 
-async function init({ budgetId, config } = {}) {
+async function init(config = {}) {
```

Comment:
`init` takes `budgetId` as an initial budget file to load; why is it being removed?

Reply:
Nevermind I see below